lang/en/bigbluebuttonbn.php

```diff
@@ -111,6 +111,8 @@
 $string['config_recording'] = 'Configuration for "Record meeting" feature';
 $string['config_recording_description'] = 'These settings are feature specific';
 $string['config_recording_default'] = 'Recording feature enabled by default';
+$string['config_oc_recording'] = 'Opencast can be used for Recording';
+$string['config_oc_recording_description'] = 'When enabled, BBB will use Opencast server for recordings. The course must use Opencast Videos Plugin (block_opencast) and have a valid Seires ID. If not, BBB uses its defaults.';
```

Comment:
As I have written in the general feedback, I propose to restrict the scope of the 'Opencast can be used for Recording' setting to just control if the Opencast series ID of the Moodle course is sent to BBB or not.

If you agree, I propose to change the language strings of this setting to something like that
```suggestion
$string['config_oc_recording'] = 'Send Opencast series ID to BBB';
$string['config_oc_recording_description'] = 'If this setting is enabled, Moodle will send the Opencast series ID of the course to BBB alongside the BBB metadata whenever a new BBB meeting is initiated. If there isn't an Opencast series ID for the course yet, a new series will be created on-the-fly. This way, Opencast can be used to process BBB recordings and to publish them in the course\'s series.';
```